edenai_apis/apis/google/google_image_api.py

```
@@ -474,3 +476,72 @@ def image__embeddings(
+            inputImage = file_wrapper.get_file_b64_content()
+        else:
+            with open(file, "rb") as image_file:
+                image_bytes = image_file.read()
+                inputImage = base64.b64encode(image_bytes).decode("utf-8")
+
+        payload = {
+            "instances": [{"image": {"bytesBase64Encoded": inputImage}}],
+            "parameters": {"dimension": embedding_dimension},
+        }
+
+        async with httpx.AsyncClient(timeout=None) as client:
```

Ruff (0.14.3)

Probable use of `httpx` call with timeout set to `None`

(S113)